src/renderer/services/priceLevel/priceLevelService.ts

```
@@ -0,0 +1,208 @@
+      // Each poll subscription completes when the API responds, so just let it finish
+      subscriptions.push(sub)
+    }
+  }
+
+  // Start polling
+  poll()
+  const intervalId = setInterval(poll, POLL_INTERVAL_MS)
+
+  const dispose = () => {
+    clearInterval(intervalId)
+    subscriptions.forEach((s) => s.unsubscribe())
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)